pkg/config/engine/containerd/config_v1.go

```diff
@@ -39,9 +39,17 @@ func (c *ConfigV1) AddRuntime(name string, path string, setAsDefault bool, confi
 
 	config.Set("version", int64(1))
 
-	if runc, ok := config.GetPath([]string{"plugins", "cri", "containerd", "runtimes", "runc"}).(*toml.Tree); ok {
-		runc, _ = toml.Load(runc.String())
-		config.SetPath([]string{"plugins", "cri", "containerd", "runtimes", name}, runc)
+	// By default we extract the runtime options from the runc settings; if this does not exist we get the options from the default runtime specified in the config.
```

Comment:
Just out of curiosity, why `runc` and then `default_runtime`?  Does it make sense to do it in the other order?

Reply:
The main motivation is to not change the existing behaviour by default.

Reply:
OK, that's fine then.